### ISSUE TITLE
How can I use the trained model for prediction? is it right to use prediction_model.predict(new_x) ?

### ISSUE THREAD

**Issue by Qianqian-Yang** [yaringal/multi-task-learning-example#17]
I'm not in the field of deep learning and computer science, but I found this work very interesting. I am confused about what should I do if I want to use the trained model for prediction?  Can I achieve this goal through prediction_model.predict(new_x)? I see only the trainable_model was trained but it can not achieve prediction. Has the prediction_model been trained at the same time? Thanks very much.

**Comment by Yhyyy-hhu** [comment]
> 

YES, the only change with the new method is on the loss function

**Comment by yyruuu** [comment]
The value I predicted with **prediction_model** is very different from the real value. Do you have any similar problems?

**Comment by YumingWu97** [comment]
just ignore it during testing.it is only used for balancing the gradients in backpropagation.
